src/atoms/CustomHtml/CustomHtml.js

```diff
@@ -1,223 +1,104 @@
-        }
-      };
-
-      window.getStreamTime = () => {
-        const trackTime = document.getElementById('trackTime');
-        const audio = document.getElementById('player');
-        const minutes = Math.floor(audio.currentTime / 60);
-        const returnedMinutes = minutes < 10 ? `0${minutes}` : `${minutes}`;
-        const seconds = Math.floor(audio.currentTime % 60);
-        const returnedSeconds = seconds < 10 ? `0${seconds}` : `${seconds}`;
-        if (trackTime) {
-          trackTime.innerHTML = `${returnedMinutes}:${returnedSeconds}`;
-        }
-      };
-    }
-
-    // If there is a script without a src, set the whole block in an iframe (or it might not work!)
-    if (
-      element.querySelector('script') &&
+      ele.querySelector('script') &&
       !hasIframe &&
-      !element.querySelector('form')
+      !ele.querySelector('form')
     ) {
-      let src = '';
-      if (whitelistRegex.test(fallback_url)) {
-        src = fallback_url;
+      let src, sourceType;
+
+      if (
+        ele.querySelector('blockquote') &&
+        ele.querySelector('.tiktok-embed')
+      ) {
+        sourceType = 'tiktok';
+      } else if (ele.querySelector('.typeform-widget')) {
+        sourceType = 'typeformWidget';
+      } else if (nodeData.attrs.fallback_url) {
+        sourceType = 'fallbackUrl';
+      } else if (ele.querySelector('script').src) {
+        sourceType = 'scriptSrc';
       }
-      safeHtml = `<iframe width="100%" height="500px" frameborder="0" scrolling="yes" marginheight="0" marginwidth="0" src="${src}">${element.innerHTML}</iframe>`;
-    }
-
-    // Enable submit button when recaptcha is successful
-    if (element.querySelector('form')) {
-      const script = document.createElement('script');
-      const inlineScript = document.createTextNode(
-        "function enableSubmit() { const subButton = document.getElementById('submitButton'); if (subButton) subButton.disabled = false; };"
-      );
-      script.appendChild(inlineScript);
-      document.head.appendChild(script);
-    }
-    return { html: safeHtml, safeScripts };
-  }
 
-  htmlStringToElement(html) {
-    if (typeof document == 'undefined') return false;
-    let template = document.createElement('div');
-    template.innerHTML = html;
-    return template;
-  }
-
-  render() {
-    if (this.props.minimal) {
-      return null;
-    }
+      switch (sourceType) {
+        case 'scriptSrc':
+          src = ele.querySelector('script').src;
+
+          break;
+        case 'fallbackUrl':
+          if (isValidUrl(nodeData.attrs.fallback_url)) {
+            src = nodeData.attrs.fallback_url;
+          }
+          break;
+        case 'typeformWidget':
+          const div = ele.querySelector('.typeform-widget');
+          if (div && div.getAttribute('data-url')) {
+            src = div.getAttribute('data-url');
+          }
+          break;
+        case 'tiktok':
+          return setState(ele.innerHTML);
```

Comment:
seems like this might not be needed - try taking out and testing a tiktok embed to see if it still works?